environment.yml

```diff
@@ -0,0 +1,25 @@
+name: nbgrader-dev
```

Comment:
That's true. How about not including it for this PR, and then make another PR with an environment file and then also update the dev installation docs telling people how they can create a new conda dev env with it?